src/components/05_pages/Permissions/Permissions.js

```diff
@@ -13,6 +13,14 @@ import {
 import Loading from '../../02_atoms/Loading/Loading';
 import { Table, TBody, THead } from '../../01_subatomics/Table/Table';
 
+export const filterPermissions = (input, permissions) =>
+  permissions.filter(
+    ({ title, description, provider, provider_label: providerLabel }) =>
+      `${title}${description}${provider}${providerLabel}`
+        .toLowerCase()
+        .includes(input.toLowerCase()),
```

Comment:
Instead of lowercasing the input on each loop wouldn't it be easier to lowercase it when it's passed in?

Reply:
For me at least encoding as much as possible within this one magic helps to understand its complete behavior (being case insensitive).